spec/models/conversation_spec.rb

```diff
@@ -110,64 +101,51 @@
       end
 
       it 'creates a message' do
-        expect{
-          Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
+        expect {
+          Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
         }.to change(Message, :count).by(1)
       end
       it 'creates a conversation' do
-        expect{
-          Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
+        expect {
+          Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
         }.to change(Conversation, :count).by(1)
       end
       it 'creates appropriate visibilities' do
-        expect{
-          Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
-        }.to change(ConversationVisibility, :count).by(@participant_ids.size)
+        expect {
+          Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
+        }.to change(ConversationVisibility, :count).by(participant_ids.size)
       end
       it 'does not save before receive' do
-        expect(Diaspora::Parser.from_xml(@xml).persisted?).to be false
+        expect(Diaspora::Parser.from_xml(xml).persisted?).to be false
       end
       it 'notifies for the message' do
         expect(Notification).to receive(:notify).once
-        Diaspora::Parser.from_xml(@xml).receive(@user1, @user2.person)
+        Diaspora::Parser.from_xml(xml).receive(user1, user2.person)
       end
     end
   end
 
   describe "#invalid parameters" do
     context "local author" do
-      before do
-        @invalid_hash = {
-          author:              peter.person,
-          participant_ids:     [peter.person.id, @user1.person.id],
-          subject:             "cool stuff",
-          messages_attributes: [{author: peter.person, text: "hey"}]
-        }
-      end
+      let(:invalid_hash) {{author: peter.person, participant_ids: [peter.person.id, user1.person.id],
+          subject: "cool stuff", messages_attributes: [{author: peter.person, text: "hey"}]}}
 
       it "is invalid with invalid recipient" do
-        conversation = Conversation.create(@invalid_hash)
-        expect(conversation).to be_invalid
+        invalid_conversation = Conversation.create(invalid_hash)
+        expect(invalid_conversation).to be_invalid
       end
     end
 
     context "remote author" do
-      before do
-        @remote_person = remote_raphael
-        @local_user = alice
-        @participant_ids = [@remote_person.id, @local_user.person.id]
-
-        @invalid_hash_remote = {
-          author:              @remote_person,
-          participant_ids:     @participant_ids,
-          subject:             "cool stuff",
-          messages_attributes: [{author: @remote_person, text: "hey"}]
-        }
-      end
+      let(:remote_person) { remote_raphael }
+      let(:local_user) { alice }
+      let(:participant_ids) { [remote_person.id, local_user.person.id] }
+      let(:invalid_hash_remote) {{author: remote_person, participant_ids: participant_ids,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.